modules/apps/store/extensions/assets/webapp/pages/assets.jag

```diff
@@ -96,7 +113,45 @@ require('/modules/store.js').exec(function (ctx) {
                 }
             });
 
-        } else {
+        }
+        else if (paging.sort == 'usage') {
```

Comment:
Please do the same changes which where mentioned in the same logic of the mobile apps.